backend/penndata/management/commands/rename_fitness_room.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management.base import BaseCommand
+
+from penndata.models import FitnessRoom
+
+
+class Command(BaseCommand):
+    help = "Renames a fitness room."
```

Comment:
Instead of having this command take in inputs, just make it apply your `cap_string` function to all the FitnessRoom objects. It's easier to deal with since we just want to run this command once and then delete it. The following method will do that:
`FitnessRoom.objects.update(name=models.functions.Func('name', function=cap_string))`